### PR TITLE
KAFKA-13671: Add ppc64le build stage

### PR DESCRIPTION
**Pull request by mimaison**
### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Comment by mimaison** [comment]
It looks like there's a permission issue with `/tmp`. I'm following up in [KAFKA-13671](https://issues.apache.org/jira/browse/KAFKA-13671).

**Comment by mimaison** [comment]
We resolved the permissions issues and it's now working. I'll let this build finish and I'll kick another one tomorrow to confirm.

**Review by mumrah — APPROVED** [review]
Thanks @mimaison, this looks good to me. Can we make the stage name something like "PPC" or "PowerPC" just for readability? 

Looking at our docs, all I could find about platform support is

> Kafka should run well on any unix system and has been tested on Linux and Solaris.

I wonder if we should include some details about the platforms we test. I also wonder about that Solaris part -- has anyone actually tested a recent build on Solaris?

Anyways, this PR LGTM pending a passing build, thanks 👍

**Comment by mimaison** [comment]
Thanks @mumrah! I've renamed the stage to `PowerPC`. 
Regarding supported platforms, I'm not aware of anybody testing recent builds on Solaris. I'll follow up with a small docs PR to update that statement.

Test failures are unrelated, merging to trunk.

File: Jenkinsfile [hunk 1]
```diff
@@ -178,6 +178,24 @@ pipeline {
...
         }
+
+        stage('PowerPC') {
+          agent { label 'ppc64le' }
+          options {
+            timeout(time: 2, unit: 'HOURS')
+            timestamps()
+          }
+          environment {
+            SCALA_VERSION=2.12
```
Comment (ijuma) [inline Jenkinsfile:189]: Scala 2.12 support has been deprecated, it would be good to use Scala 2.13 for new stages.

Comment (mimaison) [reply Jenkinsfile:189]: Good point, I'll follow up with a few more changes we should do in this area.

**Pull request merged**